pkg/webhooks/user/user.go

```diff
@@ -149,7 +149,12 @@ func (s *UserWebhook) isRedHatAssociate(userReq *userRequest) bool {
 }
 
 func (s *UserWebhook) isUsingRedHatIDP(userReq *userRequest) bool {
-	return utils.SliceContains(redHatIDP, userReq.Identities)
+	for _, idp := range userReq.Identities {
+		if strings.HasPrefix(idp, redHatIDP) {
```

Comment:
Given it's a prefix I recommend renaming args and the const to reflect that and to add the `:` into the prefix, else it will match something like `OpenShift_SRE_WasNotHere`.
https://github.com/openshift/managed-cluster-validating-webhooks/blob/909457725b7d2c177a049601c40f70fb93c6d2b3/pkg/webhooks/identity/identity.go#L21

Reply:
Discussion on slack, sounds like a simple local update is best:

```suggestion
		if strings.HasPrefix(idp+ ":", redHatIDP) {
```